tests/kafkatest/services/connect.py

```diff
@@ -331,3 +337,46 @@ def start(self):
             'tasks.max': self.tasks,
             'topics': ",".join(self.topics)
         })
+
+class MockSink(object):
+
+    def __init__(self, cc, topics, mode=None, delay_sec=10, name="mock-sink"):
+        self.cc = cc
+        self.logger = self.cc.logger
+        self.name = name
+        self.mode = mode
+        self.delay_sec = delay_sec
+        self.topics = topics
+
+    def start(self):
+        self.logger.info("Creating connector MockSinkConnector %s", self.name)
+        self.cc.create_connector({
+            'name': self.name,
+            'connector.class': 'org.apache.kafka.connect.tools.MockSinkConnector',
+            'tasks.max': 1,
+            'topics': ",".join(self.topics),
+            'mock_mode': self.mode,
+            'delay_ms': self.delay_sec * 1000
+        })
+
+class MockSource(object):
+
+    def __init__(self, cc, topics, mode=None, delay_sec=10000, name="mock-source"):
```

Comment:
Yeah, good call. By the way, I'm investigating a build failure, so hold off merging for a little bit longer.